src/DotVVM.Framework/Binding/BindingHelper.cs

```diff
@@ -19,9 +19,6 @@ public static partial class BindingHelper
     {
         public static T GetProperty<T>(this IBinding binding, ErrorHandlingMode errorMode = ErrorHandlingMode.ThrowException) => (T)binding.GetProperty(typeof(T), errorMode);
 
-        [Obsolete]
-        public static string GetKnockoutBindingExpression(this IValueBinding binding) =>
-            JavascriptTranslator.FormatKnockoutScript(binding.KnockoutExpression);
```

Comment:
This method was made obsolete in dotvvm 2.0, and I think that we should not remove, since it's used at a lot of places. Could you please add it back?

Reply:
Yes, we should definitely keep this one. Why is it obsolete? Is there any alternative?

Reply:
Yes, it's just missing a proper description... You should use `binding.GetKnockoutBindingExpression(control)`.